AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -114,6 +114,7 @@ omni.se#$#.header-banner--regular[style^="--banner-offset"] { --banner-offset: 0
 multiplayer.it#$#html { overflow: visible !important; }
 multiplayer.it#$#body #iubenda-cs-banner { display: none !important; }
 hostkey.ru###collectData
+bbc.com##header > div[class] > div[class*="-Wrapper"]
```

Comment:
this element is already blocked by 
`bbc.com##header[role="banner"] div[dir="ltr"][class*="-Wrapper "]:not([class*="-Banner"])`